circuit/src/execution-delivery/Cargo.toml

```diff
@@ -88,6 +87,7 @@ hex-literal = "0.3"
 paste = "1"
 pretty_assertions = "0.7"
 wat = "1"
+relay-substrate-client = { version = "0.1.0", path = "../../../vendor/bridges/relays/client-substrate"}
```

Comment:
missing `option = true`

Reply:
this is under dev-dependencies. Dont need have it optional there 😉